spec/features/users/03_user_signs_out_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+feature 'user signs in', %{
+  As a user
+  I want to sign out
+  So that I can securely leave the site
+} do
+
+  # Acceptance Criteria:
+  # - [ ] if I am logged in, I can log out
+  # - [ ] if I am not logged in, there is no option to log out
+  # - [ ] if I sign out, I can sign back in
```

Comment:
It's nice to put little 'x's in the brackets so that you and any others looking at the tests know that the acceptance criteria have been satisfied.